tools/applets/dds.py

```diff
@@ -1,21 +1,25 @@
-import time
 import pickle
-from PyQt5 import QtGui, QtWidgets, QtCore
+import time
+
 from artiq.applets.simple import SimpleApplet
 from jax import JaxApplet
 from jax.tools.applets.dds_channel import DDSChannel, DDSParameters
 from jax.util.ui.custom_list_widget import CustomListWidget
+from PyQt5 import QtCore, QtGui, QtWidgets
 
 
-class DDS(QtWidgets.QWidget, JaxApplet):
+class DDS(QtWidgets.QDockWidget, JaxApplet):
     # signal emitted after getting DDS parameters.
     # a signal is needed to run self.initialize_channels on the default thread.
     # widgets can only be created in the default thread.
     do_initialize = QtCore.pyqtSignal()
 
     def __init__(self, args, **kwds):
         super().__init__(**kwds)
-        self.setDisabled(True)  # start with the applet disabled, until artiq server is connected.
+        self.setObjectName("DDS")
+        self.setDisabled(
+            True
+        )  # start with the applet disabled, until artiq server is connected.
```

Comment:
this is never set back to false within the file?

Reply:
In line 58 it is set back to `False`.